common/git-hooks/pre-commit

```diff
@@ -7,4 +7,4 @@
 # are committed. The command is defined in common/config/rush/command-line.json
 # and uses the "rush-prettier" autoinstaller.
 # node common/scripts/install-run-rush.js prettier || exit $?
-node common/scripts/install-run-rush.js prettier || exit $?
+# node common/scripts/install-run-rush.js prettier || exit $?
```

Comment:
why change this?

Reply:
I feel like prettier is too opinionated and inflexible which actually uglifies many codes...

Reply:
eslint is much better